spec/policies/event_procedure_policy_spec.rb

```diff
@@ -3,75 +3,39 @@
 require "rails_helper"
 
 RSpec.describe EventProcedurePolicy do
-  describe "Scope" do
-    it "returns all event_procedures for user" do
-      user = create(:user)
-      event_procedure = create(:event_procedure, user: user)
+  let(:user) { create(:user) }
```

Comment:
Something is incorrect in this spec, I will not tell you what is, have fun finding

Reply:
Done! f9c127650d08c779d0ac2c8c204fc2c96036c006